src/test/java/org/fogbowcloud/manager/requests/api/local/http/ComputeOrdersControllerTest.java

```diff
@@ -48,132 +48,572 @@
 @PrepareForTest(ApplicationFacade.class)
 public class ComputeOrdersControllerTest {
 
-    private final String CORRECT_BODY =
+    private static final String CORRECT_BODY =
             "{\"requestingMember\":\"req-member\", \"providingMember\":\"prov-member\", "
                     + "\"publicKey\":\"pub-key\", \"vCPU\":\"2\", \"memory\":\"1024\", \"disk\":\"20\", "
                     + "\"imageName\":\"ubuntu\"}";
 
-    private final String WRONG_BODY = "";
+    private static final String WRONG_BODY = "";
+    private static final String FAKE_ORDER_ID = "fake-order-id";
 
     @Autowired
     private MockMvc mockMvc;
 
     private ApplicationFacade facade;
 
-    private final String COMPUTE_ENDPOINT = "/" + ComputeOrdersController.COMPUTE_ENDPOINT;
+    private static final String COMPUTE_ENDPOINT = "/" + ComputeOrdersController.COMPUTE_ENDPOINT;
 
     @Before
     public void setUp() throws FogbowManagerException {
-        this.facade = spy(ApplicationFacade.class);
+        this.facade = Mockito.spy(ApplicationFacade.class);
```

Comment:
This exception is never thrown. please, remove it fro the declaration

Reply:
Done